apps/hash-api/src/graphql/error.ts

```diff
@@ -0,0 +1,40 @@
+import { ApolloServerErrorCode } from "@apollo/server/errors";
+import { GraphQLError } from "graphql";
+
+export const any = (message: string, extensions?: Record<string, unknown>) =>
+  new GraphQLError(message, { extensions });
```

Comment:
Suggest we delete this to force people to pick or specify a message.

Think Apollo might default to `INTERNAL_SERVER_ERROR` here anyway? If so, might as well make people explicitly choose `internal`.

Reply:
I wasn't sure either (which is why I added `any` in the first place. I think removing it and defaulting to `internal` instead would be a good thing. Agreed. Fixed in ceca8de21a8be0c845d83cd72fbc6928fd1727db